doc/man3/SSL_CTX_set_psk_client_callback.pod

```diff
@@ -123,7 +123,9 @@ and it will use that in preference. If no such callback is present then it will
 check to see if a callback has been set via SSL_CTX_set_psk_client_callback() or
 SSL_set_psk_client_callback() and use that. In this case the B<hint> value will
 always be NULL and the handshake digest will default to SHA-256 for any returned
-PSK.
+PSK. TLSv1.3 early data exchange is possible in PSK connections only with
+B<SSL_psk_use_session_cb_func> callback, not possible with
+B<SSL_psk_client_cb_func> callback.
```

Comment:
A couple nits here: "only with the B<SSL_psk_use_session_cb_func> callback, and are not possible with the B<SSL_psk_client_cb_func> callback"

Reply:
Then I feel the start of the phrase should be `TLSv1.3 early data exchanges are`.